static/app/views/automations/components/actionFilters/constants.tsx

```diff
@@ -1,17 +1,133 @@
 import {t} from 'sentry/locale';
-import {DataConditionGroupLogicType} from 'sentry/types/workflowEngine/dataConditions';
+import {
+  DataConditionGroupLogicType,
+  DataConditionType,
+} from 'sentry/types/workflowEngine/dataConditions';
 
 export const FILTER_MATCH_OPTIONS = [
   {value: DataConditionGroupLogicType.ALL, label: t('all')},
   {value: DataConditionGroupLogicType.ANY_SHORT_CIRCUIT, label: t('any')},
   {value: DataConditionGroupLogicType.NONE, label: t('none')},
 ];
 
+export const FILTER_DATA_CONDITION_TYPES = [
+  DataConditionType.AGE_COMPARISON,
+  DataConditionType.ISSUE_OCCURRENCES,
+  DataConditionType.ASSIGNED_TO,
+  DataConditionType.ISSUE_PRIORITY_EQUALS,
+  DataConditionType.LATEST_ADOPTED_RELEASE,
+  DataConditionType.LATEST_RELEASE,
+  DataConditionType.EVENT_ATTRIBUTE,
+  DataConditionType.TAGGED_EVENT,
+  DataConditionType.LEVEL,
+];
+
+export enum MatchType {
+  CONTAINS = 'co',
+  ENDS_WITH = 'ew',
+  EQUAL = 'eq',
+  GREATER_OR_EQUAL = 'gte',
+  GREATER = 'gt',
+  IS_SET = 'is',
+  IS_IN = 'in',
+  LESS_OR_EQUAL = 'lte',
+  LESS = 'lt',
+  NOT_CONTAINS = 'nc',
+  NOT_ENDS_WITH = 'new',
+  NOT_EQUAL = 'ne',
+  NOT_SET = 'ns',
+  NOT_STARTS_WITH = 'nsw',
+  NOT_IN = 'nin',
+  STARTS_WITH = 'sw',
+}
```

Comment:
Should these be in `DataConditionType`? Looks like there's a decent amount of overlap.

Reply:
i think it should be kept separate? i think the [base conditions](https://github.com/getsentry/sentry/blob/d1b3fcb28220e3e9b978e3cd7bfc39d2b02f762f/src/sentry/workflow_engine/models/data_condition.py#L20-L27) (equal, greater than, less than, etc.) in `DataConditionType` are meant for detector triggers, but these `MatchTypes` are for the comparison data blobs